### PR TITLE
Deploy docs to github pages

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: .github/workflows/fmu-tools.yml [hunk 1]
```diff
@@ -58,3 +58,30 @@ jobs:
...
+
+      - name: Update GitHub pages
+        if: github.repository_owner == 'equinor' && github.ref == 'refs/heads/master' && matrix.python-version == '3.6'
+        run: |
+            cp -R ./build/sphinx/html ../html
+            git config --local user.email "fmu-tools-github-action"
+            git config --local user.name "fmu-tools-github-action"
+            git fetch origin gh-pages
+            git checkout --track origin/gh-pages
+            git clean -f -f -d -x
```
Comment (HansKallekleiv) [inline .github/workflows/fmu-tools.yml:78]:
```suggestion
            git clean -f -d -x
```

**Review by HansKallekleiv — APPROVED** [review]
:+1:

**Comment by berland** [comment]
Did not fully understand why I had a lot of issues with setuptools_scm not being installed in github actions. Resolved by not importing it in setup.py, but watch out for potential side-effects.

**Review by jcrivenaes — APPROVED** [review]
LGTM

**Pull request merged**